docs-private/Developer-How-To-Start.md

```diff
@@ -21,3 +21,22 @@ Others (like URL, username, password) depend on your environment.
 ```shell
 liquibase --changelog-file=./docs/db/changelog/changesets/powerauth-push-server/db.changelog-module.xml --url=jdbc:postgresql://localhost:5432/powerauth --username=powerauth status
 ``` 
+
```

Comment:
Maven has liquibase profile that process the liquibase for development.  We can reference the profile (as the profile is standard in our components)
Or we provide the generated SQL by ourselves.

I suggest not to complicate and generate SQL itself - this should be discouraged.

Reply:
This is just developer documentation in `docs-private` folder. We have it in other components, too. If you want I can remove it.